rootfs/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM quay.io/deis/base:0.3.0
+
+ENV VERSION=0.3.8.linux-amd64.go1.6.2
```

Comment:
yeah but one thing that it provides is a marker for what version someone has running in a given cluster. which is nice